test/test_socket_options.cpp

```diff
@@ -121,3 +135,114 @@ TEST_F(TestSocketOptions, LossMaxTTL)
 }
 
 
+// Try to set/get SRTO_MININPUTBW with wrong optlen
+TEST_F(TestSocketOptions, MinInputBWWrongLen)
+{
```

Comment:
This test is kinda pointless. The size check is something we need actually for all options to be done as an overall solution.

Reply:
> The size check is something we need actually for all options to be done as an overall solution.
Absolutely! Issue #1819 is created to track that.

This PR adds checks for BW-related option, as they cast to `int64_t`. The chances of mistakenly passing an `int` are higher than e.g. passing an `unit16_t` to an option expecting `int`. But all other checks are better to be added as well.